Update index.md (#1557)
nodejs/nodejs.dev: src/documentation/0045-node-folders/index.md

});
```

> **NOTE:** In Node `v16.x` the option `recursive` is **deprecated** for `fs.rmdir` of callback API, instead use the below code to delete folders that have content in them:

```js

> const fs = require('fs/promises')

Reviewer: ```suggestion
const fs = require('fs')
```

try {

>     fs.rm(dir, { recursive: true, force: true })

Reviewer: Since `fs.rm` is a promise here, it'll go to `UnhandledPromiseRejectionWarning` instead of getting caught into the catch block.

You can use the normal `fs` module here, to maintain consistency with the snippet before this

Author: nice catch, thanks 😅

});
```

> > **NOTE:** In Node `v16.5.0` the option `recursive` is **deprecated** for `fs.rmdir` of callback API, instead use the below code to delete folders that have content in them:

Reviewer 2: ```suggestion
> **NOTE:** In Node `v16.x` the option `recursive` is **deprecated** for `fs.rmdir` of callback API, instead use the below code to delete folders that have content in them:
```

```js
const fs = require('fs/promises')

> try{

Reviewer 2: ```suggestion
try {
```

> fs.rm(dir,{recursive:true,force:true})

Reviewer 2: ```suggestion
    fs.rm(dir, { recursive: true, force: true })
```

> }

Reviewer 2: ```suggestion

```

> catch(err){

Reviewer 2: ```suggestion
} catch (err) {
```

> console.log(err)

Reviewer 2: ```suggestion
    console.log(err)
```
